Clinical-Genomics/chanjo2
# Temp fix the d4tools super high region coverage bug #78

### This PR adds | fixes:
- Temp fix #299 

### How to test:
- All automatic tests should pass
- Deploy on stage and check coverage over real gene panels for a scout case

### Expected outcome:
- [ ] Coverage stats should be within the expected range

### Review:

This [version](https://semver.org/) is a:
